### PR TITLE
Added more servers!

### PR DESCRIPTION
**Pull request by ahmed605**
Changes:

- Added French Person, Animal, Object servers

- Fix Arabic server doesn't work in some countries

- Fixed Russian person server

### CODE REVIEW

File: Akinator.Api.Net.Tests/ClientIntegrationTests.cs [hunk 1]
```diff
@@ -351,5 +351,86 @@ public async Task ReuseSessionWorks()
                 Assert.AreEqual(2, question.Step);
             }
         }
+      
+      [TestMethod]
```
Comment (janniksam) [inline Akinator.Api.Net.Tests/ClientIntegrationTests.cs]: missing a space

Comment (janniksam) [reply Akinator.Api.Net.Tests/ClientIntegrationTests.cs]: I have sorted the test methods alphanumerically. Can you move it infront of the german tests? Also use "Frensh", not "France" in the name. The language is frensh, France is just the country :)

File: Akinator.Api.Net/Enumerations/Language.cs [hunk 2]
```diff
@@ -8,5 +8,6 @@ public enum Language
         Italian,
         Spanish,
         Russian,
+        France,
```
Comment (janniksam) [inline Akinator.Api.Net/Enumerations/Language.cs]: like i said before, Frensh, not France :)

File: Akinator.Api.Net/Utils/ServerSelector.cs [hunk 3]
```diff
@@ -29,7 +29,7 @@ public static class ServerSelector
                 {
                     Language.Arabic, new[]
                     {
-                        new Server(ServerType.Person, "srv12.akinator.com:9187/ws")
+                        new Server(ServerType.Person, "srv2.akinator.com:9155/ws")
```
Comment (janniksam) [inline Akinator.Api.Net/Utils/ServerSelector.cs:32]: Why did you change this? Did the server change or was it the wrong type? It worked yesterday for me.

File: Akinator.Api.Net/Utils/ServerSelector.cs [hunk 4]
```diff
@@ -46,10 +46,18 @@ public static class ServerSelector
...
+                        new Server(ServerType.Person, "srv3.akinator.com:9217/ws"),
+                        new Server(ServerType.Object, "srv3.akinator.com:9218/ws"),
+                        new Server(ServerType.Animal, "srv3.akinator.com:9259/ws")
+                    }
+                },
                 {
                     Language.Russian, new[]
                     {
-                        new Server(ServerType.Person, "srv14.akinator.com:9286/ws")
+                        new Server(ServerType.Person, "srv12.akinator.com:9190/ws")
```
Comment (janniksam) [inline Akinator.Api.Net/Utils/ServerSelector.cs:60]: why did you change this?

**Review by janniksam — CHANGES_REQUESTED** [review]

File: Akinator.Api.Net/Utils/ServerSelector.cs (hunk 4 above)
Comment (ahmed605) [reply Akinator.Api.Net/Utils/ServerSelector.cs:60]: I changed this because the previous servers go down a lot of times and return KO - Server Down

File: Akinator.Api.Net/Utils/ServerSelector.cs (hunk 3 above)
Comment (ahmed605) [reply Akinator.Api.Net/Utils/ServerSelector.cs:32]: because this server contain limited characters and doesn't work in every country (doesn't work for me in Egypt (it works when I'm using vpn))

**Review by janniksam — APPROVED** [review]
